config/rbac/kustomization.yaml

```diff
@@ -13,6 +13,6 @@ resources:
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
 # - auth_proxy_service.yaml
```

Comment:
Isn't this something that we want too (with an updated `.spec.ports[0].targetaPort`.) ?

Reply:
Yes, we can include this svc for the kube-rbac-proxy indeed. Then, I will not need to create a svc for the Kubestellar controller manager in the scripts to configure the KS monitoring tools.

Reply:
I have pushed a new commit to uncomment the svc line